### PR TITLE
REPLICA: PR #355

### PR DESCRIPTION
**Pull request by mmcky**
This is a replicate of #355

### CODE REVIEW

File: lectures/lake_model.md [hunk 1]
```diff
@@ -265,12 +265,8 @@ class LakeModel:
         --------
         xbar : steady state vector of employment and unemployment rates
         """
-        x = np.full(2, 0.5)
-        error = tol + 1
-        while error > tol:
-            new_x = self.A_hat @ x
-            error = np.max(np.abs(new_x - x))
-            x = new_x
+        x = np.array([self.A_hat[1, 0], self.A_hat[0, 1]])
```
Comment (shlff) [inline lectures/lake_model.md]:
```suggestion
        x = np.array([self.A_hat[0, 1], self.A_hat[1, 0]])
```

Please find experiment:

https://gist.github.com/shlff/02f5aac35f8793cdf08935e13174b131

where method with ``x = np.array([self.A_hat[0, 1], self.A_hat[1, 0]])`` generates the same result as the original iterative method.

File: lectures/lake_model.md [hunk 2]
```diff
@@ -1016,12 +1013,8 @@ class LakeModelModified:
...
```
Comment (shlff) [inline lectures/lake_model.md]:
```suggestion
        x = np.array([self.A_hat[0, 1], self.A_hat[1, 0]])
```

Same here.

**Review by shlff — CHANGES_REQUESTED** [review]
Thanks @mmcky . Hi @jstac and @oyamad , please find my reviews here.

**Comment by mmcky** [comment]
Closing as no longer required

**Pull request closed without merging**